modules/azurerm_postgresql_flex/variables.tf

```diff
@@ -56,10 +56,10 @@ variable "server_version" {
   default     = "13"
 }
 
-variable "public_network_access_enabled" {
-  description = "Whether or not public network access is allowed for this server. Defaults to true"
-  type        = bool
-  default     = true
+variable "connectivity_method" {
+  description = "Network connectivity options to connect to your flexible server. Valid options are 'public' and 'private'. Defaults to public"
+  type        = string
+  default     = "public"
 }
```

Comment:
Was this every surfaced to the end user? If not no comment here. But if it was the change here will be a breaking change.

Reply:
No this was never surfaced to end user before, it was an internal variable.
Previously only public access was available so no inputs were required from User. Now with this new change User will have the ability to change the connectivity method, it defaults to public access.